CommMicro.py

```diff
@@ -221,25 +245,26 @@ def setGOVal(self,tPlot,bPlot):
         # get LASTPATH from getUserVal()
         makedirs(LASTPATH, exist_ok=True)
 
-        # This kinda cheats... really just ceil of timMeas/8
-        #numbWaveF= str(timMeas//8 +int(timMeas % 8 > 0))
-        numbWaveF = str(math.ceil(timMeas/8))
+        numbWaveF = str(self.ui.spinBox_buffers.value())
+
+        wsp = str(self.ui.comboBox_decimation.currentText())

```

Comment:
What does wsp stand for?

Reply:
It is one of the input parameters to the acquisition script, controlling the data decimation. In the end the default sampling rate is divided by this factor. We'd have to ask Sergio and co where the name comes from. The most detailed I've seen is "wave_samp_per" in the header of the output file.

Reply:
I've made the variable name more descriptive.